src/activities/components/ActivityItem.vue

```diff
@@ -181,28 +222,36 @@ export default {
 
 .content
   width 100%
-  font-size .8em
-  cursor pointer
   transition background-color 2s ease
 
-  .featured-text
-    display inline
-    margin-right .3em
-    font-size 1.5em
-
-  &.isEmpty:not(.isDisabled)
-    background repeating-linear-gradient(
-      135deg,
-      white,
-      white 15px,
-      $lightRed 15px,
-      $lightRed 30px
-    )
-
   &.isUserMember
     &:not(.isDisabled)
       background linear-gradient(to right, $lightGreen, $lighterGreen)
 
   &.isDisabled
     background $lightRed
+
+  .content-inner
+    width 100%
+    padding 12px
+
+    .featured-text
+      display inline
+      margin-right .3em
+
+.conversation-section
+  font-weight 500
+  color $secondary
+  box-shadow 0 -1px 0 rgba(0, 0, 0, 0.06)
+
+.q-btn.open-conversation-button
+  ::v-deep .q-btn__wrapper
```

Comment:
Minor style: we have used `>>>` elsewhere which works as an alias for `::v-deep`

Reply:
Ah, in our project at work `>>>` doesn't work. Changed it.